src/ontology/fbbt-edit.obo

```diff
@@ -39459,7 +39455,6 @@ synonym: "DRA R8" EXACT [FlyBase:FBrf0243430]
 xref: VFB:FBbt_00004209
 intersection_of: FBbt:00004227 ! photoreceptor cell R8
 intersection_of: BFO:0000050 FBbt:02000000 ! part_of dorsal margin ommatidium
-relationship: BFO:0000050 FBbt:00052136 ! part_of dorsal rim area optic column
```

Comment:
This is somewhere that a 'synapses restricted to region' relationship would be nice. For now maybe change to 'has synaptic IO in region'.